eo-runtime/src/test/eo/org/eolang/bytes-tests.eo

```diff
@@ -342,3 +342,9 @@
       $.equal-to
         "hello world"
 
+[] > bytes-hashes-equals
```

Comment:
@Graur good test, but a weak one. I suggest adding one more test, which would compare hash with a value we expect.

Reply:
@yegor256 Agree. I will add a new test, thanks!